manifests/4.11/manifests/nfd.clusterserviceversion.yaml

```diff
@@ -797,7 +797,6 @@ spec:
                 - name: NODE_FEATURE_DISCOVERY_IMAGE
                   value: quay.io/openshift/origin-node-feature-discovery:4.10
                 image: quay.io/openshift/origin-cluster-nfd-operator:4.10
-                imagePullPolicy: Always
```

Comment:
This I don't  know... interesting, going to ask the operator-fw team before moving with this change, I think is expected to have this

Reply:
It is the CSV one, the one that the OLM deploys, and the one that is truly used to install the deployment.